test/e2e/kubevirt.go

```diff
@@ -972,17 +1002,35 @@ passwd:
 			waitForPodsCondition(httpServerTestPods, conditionFn)
 			httpServerTestPods = updatePods(httpServerTestPods)
 		}
+
+		removeImagesInNode = func(node, imageURL string) {
```

Comment:
return an error on this function and catch it at the caller, 

Also maybe is worth it to have a 
```golang
removeImagesInNodes(imageURL string) error {
nodesList, err := fr.ClientSet.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
			if err != nil {
                           return err
                        }
			for _, node := range nodesList.Items {
				if err := removeImagesInNode(node.Name, strings.Split(imageURL, ":")[0]); err != nil {
                                   return err
                                }
			}
                        return nill
}
```

Reply:
[DONE](https://github.com/ovn-kubernetes/ovn-kubernetes/compare/8964727008d70bb47d29ee1807f8584cac23eec4..72eb4f766ab594ef05468d9f154421b1a56980ca)